spec/services/duplicate_profile_checker_spec.rb

```diff
@@ -27,73 +26,168 @@
       session[:encrypted_profiles] = {
         profile.id.to_s => SessionEncryptor.new.kms_encrypt(active_pii.to_json),
       }
+      stub_analytics
     end
 
     context 'when user has active IAL2 profile' do
-      context 'when user has not been checked for duplicate profile' do
-        context 'when user does not have other accounts with matching profile' do
-          let(:user2) { create(:user, :proofed_with_selfie) }
-
-          it 'does not create a new duplicate profile confirmation' do
-            dupe_profile_checker = DuplicateProfileChecker.new(
-              user: user,
-              user_session: session,
-              sp: sp,
-            )
-            dupe_profile_checker.check_for_duplicate_profiles
-            dupe_profile_object = DuplicateProfile.involving_profile(
-              profile_id: profile.id,
-              service_provider: sp.issuer,
-            )
-            expect(dupe_profile_object).to eq(nil)
-          end
+      context 'when user does not have other accounts with matching profile' do
```

Comment:
It would be more appropriate to say "when there are no other matching profiles". Similarly in other places.